### PR TITLE
[MSKINS-239] Drop support for defunct Flattr.com

### PR DESCRIPTION
**Pull request by slachiewicz**

### CODE REVIEW

**Review by michael-o — CHANGES_REQUESTED** [review]
You missed all of these copy goals: https://github.com/apache/maven-fluido-skin/blob/e33c1b6448674fc3356112728dd2cc6406274961/pom.xml#L641

**Comment by slachiewicz** [comment]
Thx, fixed

File: src/site/apt/index.apt.vm [hunk 1]
```diff
@@ -436,32 +436,7 @@ Welcome to ${project.name}!
...
-          <style>compact</style>
-      </flattr>
-    </fluidoSkin>
-  </custom>
-  [...]
-</project>
-+-----+
-
- By default, it is rendered in the sidebar, if the topbar only is available it will be rendered there.
+ Since version <<2.0.0-M9>>, fluido-skin supports for Flatter buttons was dropped.
```
Comment (michael-o) [inline src/site/apt/index.apt.vm:439]: Flattr

**Review by michael-o — CHANGES_REQUESTED** [review]
Typo

**Review by michael-o — APPROVED** [review]

**Pull request merged**